gitpod-io/website
# chore: Remove Subjects from `/contact/sales` form

## Description

Removed Subject field from `/contact/sales` form. On Front, Segment, we would still receive `Subject` as it will be passed on the URL parameter (`?subject=enterprise` OR `?subject=Dedicated%20self-serve` -- by default it's `enterprise`).

## Related Issue(s)



Fixes MKWEB-87

## How to test



- Open preview
- Go to pricing page
- Click `Talk to sales` under Gitpod Cloud & `Request template` under Gitpod Dedicated. 
- Try to fill out the sales forms & then check front (you will still be able to see the subject category there).

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing


## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies, data storing, or analytics.


</details>
